pcp/Platform.c

```diff
@@ -0,0 +1,874 @@
+/*
+htop - linux/Platform.c
+(C) 2014 Hisham H. Muhammad
+(C) 2020-2021 htop dev team
+(C) 2020-2021 Red Hat, Inc.  All Rights Reserved.
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h"
+
+#include <math.h>
+
+#include "BatteryMeter.h"
+#include "ClockMeter.h"
+#include "Compat.h"
+#include "CPUMeter.h"
+#include "DateMeter.h"
+#include "DateTimeMeter.h"
+#include "DiskIOMeter.h"
+#include "HostnameMeter.h"
+#include "LoadAverageMeter.h"
+#include "Macros.h"
+#include "MainPanel.h"
+#include "MemoryMeter.h"
+#include "Meter.h"
+#include "NetworkIOMeter.h"
+#include "Object.h"
+#include "Panel.h"
+#include "PCPProcess.h"
+#include "PCPProcessList.h"
+#include "Platform.h"
```

Comment:
`pcp/Platform.h` should be first include after `config.h`. Cf. style guide.

Reply:
Yep re first header - we seem to be a bit inconsistent about the relative path though, e.g. linux/Platform.c does not include the linux/ prefix.

Reply:
That inconsistency should have been fixed as part of #616 recently.

Reply:
+1 ... I'll update the PCP code shortly to follow that convention.